CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ All notable changes for each version of this project will be documented in this
     - **Feature** advanced keyboard navigation support has been added. Read up more information in the [ReadMe](https://github.com/IgniteUI/igniteui-angular/tree/master/projects/igniteui-angular/src/lib/calendar/README.md)
 
 - **New component** `IgxMonthPicker`:
-    - Provides the ability to pick a specific month. Read up more information in the [ReadMe](https://github.com/IgniteUI/igniteui-angular/tree/master/projects/igniteui-angular/src/lib/month-picker/README.md)
+    - Provides the ability to pick a specific month. Read up more information in the [ReadMe](https://github.com/IgniteUI/igniteui-angular/tree/master/projects/igniteui-angular/src/lib/calendar/month-picker/README.md)
```

Comment:
Month picker is a separate component similar to the date picker. Is there a reason why it's located under calendar folder?

Reply:
It shares most of the code.